### PR TITLE
CORTX-31029: Port  number added in curl command to validate CSM API

### PR DESCRIPTION
**Pull request by mukul-seagate11**
Signed-off-by: Mukul Malhotra <mukul.malhotra@seagate.com>

### Describe your changes in brief
Port  number added in curl command to validate CSM API

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [x] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

### CODE REVIEW

**Review by rshenoy0831 — CHANGES_REQUESTED** [review]
One Minor Change is Required

curl --request POST '<https://$HOSTNAME:31169/api/v2/login?debug'

closing > bracket is missing after $HOSTNAME

**Comment by mukul-seagate11** [comment]
> One Minor Change is Required
> 
> curl --request POST '<https://$HOSTNAME:31169/api/v2/login?debug'
> 
> closing > bracket is missing after $HOSTNAME

Done

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 1]
```diff
@@ -65,14 +65,18 @@ Procedure
...
-**********
-#. Validate the CSM endpoint using below command. Use hostname or ip of your OVA VM.
+**********************************
+
+#. Validate the CSM endpoint using below command. Use hostname or ipaddress as per your VM
    
    ::
    
-        curl --request POST '<https://$HOSTNAME OR https://$IPADDRESS>/api/v2/login?debug' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
+      curl --request POST '<https://$HOSTNAME:31169/api/v2/login?debug>' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
```
Comment (gauravchaudhari02) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:76]: is it going to be a fix port each time?

**Review by rshenoy0831 — APPROVED** [review]
Looks Good

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 1 above)
Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:76]: No

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**